sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ResourceGroupOperations.cs

```diff
@@ -578,10 +582,12 @@ public virtual async Task<ResourceGroupCreateOrUpdateOperation> StartSetTagsAsyn
 
             try
             {
-                ResourceGroup resource = await GetResourceAsync(cancellationToken).ConfigureAwait(false);
-                var patch = new ResourceGroupPatchable();
-                patch.Tags.ReplaceWith(tags);
-                var originalResponse = await RestClient.UpdateAsync(Id.Name, patch, cancellationToken).ConfigureAwait(false);
+                var tagOperations = GetTagOperations();
```

Comment:
Given that we are switching to the TagsRestOperations there are no LRO methods in there, so we don't need the LRO variants here anymore.  They were here because when we were using Update as the mechanism to change tags some resources had LRO update and some didn't so we were going for consistency, now we can just have the non LRO variants

Reply:
Done